specs/signature-specification.md

```diff
@@ -127,43 +84,44 @@ A signature envelope consists of the following components:
 
-Notary v2 supports the following envelope formats:
+The Notary Project signature supports the following envelope formats:
 
 - [JWS](./signature-envelope-jws.md)
-- [COSE Sign1](./signature-envelope-cose.md)
+- [COSE](./signature-envelope-cose.md)
 
 ### Payload
 
-Notary v2 payload is a JSON document with media type `application/vnd.cncf.notary.payload.v1+json` and has following properties.
+The Notary Project signature payload is a JSON document with media type `application/vnd.cncf.notary.payload.v1+json` and has following properties.
 
 - `targetArtifact` : Required property whose value is the descriptor of the target artifact manifest that is being signed. Only [OCI descriptor][oci-descriptor] is supported.
-  - Descriptor MUST contain `mediaType`, `digest`, `size` fields.
-  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules]. Notary v2 uses annotations for storing both Notary specific and user defined metadata. The prefix `io.cncf.notary` in annotation keys is reserved for use in Notary v2 and MUST NOT be used outside this specification.
-  - Descriptor MAY contain `artifactType` field for artifact manifests, or the `config.mediaType` for `oci.image` based manifests.
+  - Descriptor MUST contain `mediaType`, `digest`, and `size` fields.
+  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules]. The Notary Project signature uses annotations for storing both Notary Project specific and user defined metadata. The prefix `io.cncf.notary` in annotation keys is reserved for use in Notary Project signature and MUST NOT be used outside this specification.
+  - Descriptor MAY contain `artifactType` field for the `config.mediaType` of OCI image manifests.
 
 #### Examples
 
 ```jsonc
 {
   "targetArtifact": {
-    "mediaType": "application/vnd.oci.image.manifest.v1+json",
-    "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
-    "size": 16724,
-    "annotations": {
-        "io.wabbit-networks.buildId": "123"  // user defined metadata
-    }
+    "mediaType": "sbom/example",
+    "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
+    "size": 32654
   }
 }
 ```
 
 ```jsonc
 {
   "targetArtifact": {
-    "mediaType": "sbom/example",
-    "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
-    "size": 32654
+    "mediaType": "application/vnd.oci.image.manifest.v1+json",
+    "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
+    "size": 16724,
+    "artifactType": "application/vnd.cncf.notary.signature",
```

Comment:
Why is the `artifactType` of the target artifact is a `notary.signature`? 
```suggestion
```

Reply:
There are two examples in this spec. One of them contains `artifactType` in the descriptor if implementations choose to use `artifactType` in descriptor. It is an optional field for `Descriptor`.